### PR TITLE
Dependency upgrades

### PR DESCRIPTION
**Pull request by 0xdade**
Initial testing seems like these work just as fine as they did with the older versions, however I'm unable to say with certainty until I test a little further. Opening as a draft pull request to mark that it's being worked on and also to see if any of the automatic checks / docker build runs automatically against draft PRs.

### CODE REVIEW

File: natlas-agent/requirements.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-certifi==2018.11.29
+certifi==2020.4.5.2
 chardet==3.0.4
-idna==2.8
-python-dotenv==0.10.1
+idna==2.9
+python-dotenv==0.13.0
 python-libnmap==0.7.0
-requests==2.21.0
-sentry-sdk[flask]==0.10.2
```
Comment (ajacques) [inline natlas-agent/requirements.txt:7]: Thanks!

File: natlas-server/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM node:10-jessie as webpack
+FROM node:12-stretch as webpack
```
Comment (ajacques) [inline natlas-server/Dockerfile:1]: The docker-compose.yml needs to be updated with this new version.

**Comment by 0xdade** [comment]
Upgrading agent to python:3.8 and ubuntu:20.04 doesn't seem to be feasible without changing how chromium is downloaded, because docker build cannot talk to the snap store which is where the chromium deb appears to point:

```
Selecting previously unselected package chromium-browser.
(Reading database ... 6499 files and directories currently installed.)
Preparing to unpack .../00-chromium-browser_81.0.4044.129-0ubuntu0.20.04.1_amd64.deb ...
=> Installing the chromium snap
==> Checking connectivity with the snap store
===> Unable to contact the store, trying every minute for the next 30 minutes
```

**Comment by 0xdade** [comment]
I went ahead and marked this as ready for review. I think it's worth doing these upgrades now as we approach the api rewrite and there are a bunch of useful changes in these updates. The agent will have to stay on 18.04 ubuntu for now until we find a solution for the chromium snap build problem.

**Pull request merged**